src/prepareRequest.js

```diff
@@ -28,7 +28,9 @@ function prepareRequest (data) {
         break
 
         case 'initSessionByUserToken': 
-            url = `${url}/initSession?user_token=${data.userToken}`
+            queryString = `?user_token=${data.userToken}&`
```

Comment:
Hi, @Gianfranco97 
The `user_token` as `queryString `  is deprecated.
You should pass this parameter in 'Authorisation' HTTP header